pkg/common-controller/snapshot_controller.go

```diff
@@ -571,6 +571,9 @@ func (ctrl *csiSnapshotCommonController) createSnapshotContent(snapshot *crdv1.V
 		return nil, newControllerUpdateError(utils.SnapshotKey(snapshot), err.Error())
 	}
 
+	msg := fmt.Sprintf("Waiting for a snapshot %s to be created by the CSI driver.", snapshot.Name)
```

Comment:
Replace snapshot.Name with utils.SnapshotKey(snapshot).  This will print out namespace/name of the snapshot.